internal/cluster/transition.go

```diff
@@ -143,7 +144,7 @@ func (th *transitionHandler) PostPrepareForInstallation(sw stateswitch.StateSwit
 	if !ok {
 		return errors.New("PostPrepareForInstallation invalid argument")
 	}
-	extra := append(append(make([]interface{}, 0), "install_started_at", strfmt.DateTime(time.Now()), "installation_preparation_completion_status", ""), resetLogsField...)
+	extra := append(append(make([]interface{}, 0), "install_started_at", strfmt.DateTime(time.Now())), resetLogsField...)
```

Comment:
?

Reply:
This is to prevent that field being wiped on restart of installation.
If this remains here, we cannot use this field as the basis of any condition that will persist across installation attempts.